tests/test_modeling_common.py

```diff
@@ -130,15 +130,13 @@ def test_attention_outputs(self):
             encoder_seq_length = encoder_seq_length * self.model_tester.num_hashes
 
         for model_class in self.all_model_classes:
-            config.output_attentions = True
```

Comment:
Basically, in every test which sets `config.output_attentions = True` we should instead add this to the forward function to have the same result.